lib/path.cpp

```diff
@@ -266,6 +267,13 @@ bool Path::fileExists(const std::string &file)
     return f.is_open();
 }
 
+
+bool Path::directoryExists(const std::string &path)
+{
+    struct stat info;
+    return stat(path.c_str(), &info) == 0 && (info.st_mode & S_IFDIR);
```

Comment:
I think this might fail in case it is a symlink. We might have that issue in other places but I don't think we use anywhere else (didn't check).

Maybe add a test to `test/cli/test-other.py` which generates a path with `os.symlink()` to be sure.

Reply:
`stat()` follows symlinks, `lstat()` does not. I will add UT, though.

Reply:
Ah. I somehow overlooked that in the manpage. So no need to add that test then.

Reply:
Ok, no UT needed then. Resolving this thread.